public/data/PeroHub.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Peter Ime",
+  "type": "personal",
+  "bio": "Hi there! I'm a frontend developer with deeper passion to build for Open Source the new sauce!",
+  "avatar": "https://github.com/PeroHub.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/PeroHub",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Wuzpa",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Data Structures & Algorithms",
+      "date": "January 2022",
+      "icon": "Freecodecamp",
```

Comment:
```suggestion
      "icon": "freecodecamp",
```

Reply:
I have made changes to this request also.